src/gsw_internal_funcs/mod.rs

```diff
@@ -1224,7 +1224,55 @@ gsw_linear_interp_SA_CT
 gsw_pchip_interp_SA_CT
 gsw_rr68_interp_SA_CT
 gsw_spline_interp_SA_CT
-gsw_gibbs_pt0_pt0
 gsw_gibbs_ice_part_t
 gsw_gibbs_ice_pt0
 */
+
+
```

Comment:
We need some minimum documentation. At least:
- One line short description;
- `Arguments`;
- `Returns`;

The module `volume` has a few examples: https://github.com/castelao/GSW-rs/blob/main/src/volume.rs